ModelController.js

```diff
@@ -8,7 +8,7 @@ const path = require('path');
 
 const CONFIG_FILE = path.join(__dirname, 'config.json');
 
-var modelConfig = util.loadJsonFile(CONFIG_FILE);
+var modelConfig = {};// util.loadJsonFile(CONFIG_FILE);
```

Comment:
No need to leave the old line as a comment

Reply:
Well the old line just needed to check if the file existed, otherwise it was useful. Its been updated now (locally, pushed soon)